src/css/styles.scss

```diff
@@ -93,23 +103,23 @@ img {
 
 .you-vs-world {
   text-align: center;
-}
 
-.you-vs-world h4 {
-  margin-bottom: 4px;
-  margin-top: 4px;
-}
+  h3 {
+    margin-bottom: .25rem;
+    margin-top: .25rem;
+  }
```

Comment:
All of the changes in this file and the next are unrelated to the title of the PR. Remember to either keep your PRed commits specific to the topic of the PR, or give more context on what changes this PR will apply